src/TreeTenantCounter/App.svelte

```diff
@@ -66,7 +66,7 @@
               {language[locale].treesPlanted}
             </p>
           </div>
-          {#if goal !== 0}
+          {#if goal != 0 && goal != "" && !isNaN(goal) }
```

Comment:
We have similar code in our other widgets, e.g. at https://github.com/Plant-for-the-Planet-org/planet-widgets/blob/9be970ca0f5fae15da89e61715188e7b1e948432/src/TreeMap/App.svelte#L223 or at https://github.com/Plant-for-the-Planet-org/planet-widgets/blob/9be970ca0f5fae15da89e61715188e7b1e948432/src/TreeProfile/App.svelte#L91 

I would like to have a general solution which is correct for all widgets, so can you once more test if changing the check from `goal !== 0` towards `goal != 0` is good enough in this case or if we really need to add the additional checks `&& goal != "" && !isNaN(goal)` as well and if they are missing in the other widgets then?

Reply:
I tested with `{#if goal != 0}` and could not produce a case showing `NaN` yet.

Reply:
As this seems to be stuck, I changed it myself.